examples/dataflow-scala-kafka2avro/src/main/scala/utils/Utils.scala

```diff
@@ -0,0 +1,88 @@
+/**
+  Copyright 2019 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  */
+
+package com.google.cloud.pso.kafka2avro.utils
+
+import com.sksamuel.avro4s.{ AvroSchema, SchemaFor }
+import java.io.{ ByteArrayInputStream, ObjectInputStream, ObjectOutputStream }
+import java.util.Base64
+import org.apache.avro.{ Schema, SchemaBuilder }
+import org.apache.commons.io.output.ByteArrayOutputStream
+import org.slf4j.Logger
+import pureconfig.error.ConfigReaderFailures
+
+
+object Kafka2AvroUtils {
+  /**
+    * Returns an object of type T from a base64 encoded string.
+    *
+    * Overwrite this function for your deserialization code. In this example, we
+    * assume that Kafka contains plain old Java objects, encoded as base64
+    * strings.
+    *
+    * @tparam T the type of the object to be recovered
+    * @param s string with a base64 encoded serialized object
+    */
+  def string2object[T](s: String): T = {
+    val data: Array[Byte] = Base64.getDecoder.decode(s)
+    val ois: ObjectInputStream = new ObjectInputStream(new ByteArrayInputStream(data))
```

Comment:
It might be better to move this to a separate method that can use try-with-resources to close the stream and also handle any exceptions